primefaces/src/main/resources/org/primefaces/Messages.properties

```diff
@@ -13,7 +13,7 @@ primefaces.datatable.SORT_ASC = Ascending
 primefaces.datatable.SORT_DESC = Descending
 primefaces.datatable.SORT_LABEL = Sort
 primefaces.dialog.aria.CLOSE = Close
-primefaces.fileupload.CLAM_AV_FILE = ClamAV has identified file '{0}' as potentially dangerous with message '{1}'
```

Comment:
Don't we need to do this in all message bundles?

Reply:
Extended now whatsnew.md and also did some more fixes in the other messages.properties files